tests/unit/einstein/test_operations.py

```diff
@@ -74,18 +87,21 @@ def tearDown(self):
 
 
 class TestCreateSnapshot(TestCase):
+    """
+    Provisions an imported image and creates snapshot
```

Comment:
Can you please put ``TestCreateSnapshot``, ``TestListSnapshots``, ``TestRemoveSnapshot`` in one class called  ``TestSnapshots`` with methods ``test_create_snapshot``, ``test_list_snapshots``, ``test_remove_snapshot``. There is a huge amount of repetition in ``setUp``.